app/controllers/api/api_controller.rb

```diff
@@ -2,9 +2,9 @@ module Api
   class ApiController < Api::BaseController
     def index
       res = {
-        :name        => @name,
-        :description => @description,
-        :version     => @version,
+        :name        => Settings.base.name,
```

Comment:
In hindsight, I'm finding this Settings here to be very confusing.  It *feels* like it should be resolving to the global Settings, but it's not because of the reuse of the name.  I'm wondering if it would be better to call it ApiSettings.  Although, I admit that Api::ApiSettings is redundant looking, you are never going to use it that way.